### PR TITLE
Updated motion list, form and detail views.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Ready for first review.

### CODE REVIEW

File: openslides/motions/static/templates/motions/motion-form.html [hunk 1]
```diff
@@ -29,15 +33,17 @@ <h1 ng-if="!motion.id" translate>New motion</h1>
...
       </ui-select-choices>
-    </ui-select>
+    </ui-select>-->
   </div>
-  <div class="form-group" ng-class="{'has-error has-feedback': motionForm.inputTitle.$error.required}">
+  <div class="form-group">
     <label for="inputName" translate>Title</label>
-    <input type="text" ng-model="motion.title" class="form-control" name="inputTitle" ng-required="true">
+    <!-- TODO: get title from latest version for edit form (ng-value does not work) -->
+    <input type="text" ng-model="motion.title" ng-value="motion.versions.slice(-1)[0].title" class="form-control" name="inputTitle" required>
```
Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-form.html]: @ostcar Here I need your help. In the motion-edit view the title and text fields are empty. How can I show the values of motion.versions without changing the ng-model (which is required for setting the form data).

File: openslides/motions/static/templates/motions/motion-list.html [hunk 2]
```diff
@@ -32,11 +32,27 @@ <h1 translate>Motions</h1>
 <table class="table table-striped table-bordered table-hover">
   <thead>
     <tr>
-      <th ng-click="toggleSort('identifier')" class="sortable">
+      <!-- TODO: Add motion.agenda_item to rest api -->
```
Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-list.html]: @normanjaeckel I still missing motion.agenda_item in the rest api. Can you add this reference please?

Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-list.html]: Issue created:  #1559

**Comment by emanuelschuetze** [comment]
All done. Ready for last review.

**Comment by normanjaeckel** [comment]
Ok (untested).

**Pull request merged**